synapse/handlers/room.py

```diff
@@ -750,7 +750,14 @@ def send(etype, content, **kwargs):
                     power_level_content["users"][invitee] = 100
 
             if power_level_content_override:
-                power_level_content.update(power_level_content_override)
+                power_level_content = power_level_content_override
```

Comment:
I'm not sure this is correct. The spec isn't entirely clear, but says:

> This object is applied on top of the generated m.room.power_levels

which to me sounds more like a merge than a replacement?

Reply:
https://github.com/matrix-org/synapse/issues/5579#issuecomment-509029439

Reply:
ok, let's take the discussion there.

Reply:
@t3chguy can you update this to just fix the initial problem in #5579 (the broken room) rather than have it get lost in the weeds about whether `power_level_content_override` replaces or merges?